pytket/docs/backends.rst

```diff
@@ -5,7 +5,7 @@ Contains :py:class:`Backend` abstract class and associated methods. In pytket a
 
 There are several `example notebooks <https://tket.quantinuum.com/examples>`_ on pytket :py:class:`Backend`\s. If you are interested in developing your own :py:class:`Backend` or pytket extension then see the `creating backends <https://tket.quantinuum.com/examples/creating_backends.html>`_ tutorial.
 
-Notebook tutorials specific to the :py:class:`QuantinuumBackend` can be found `here <https://github.com/CQCL/pytket-quantinuum/tree/develop/examples>`_.
+Notebook tutorials specific to the :py:class:`QuantinuumBackend` can be found `here <https://github.com/CQCL/pytket-quantinuum/tree/main/examples>`_.
```

Comment:
Is this change intended? We have not updated that repo yet?

Reply:
You're right, that was a mistake, I'll revert it.